docs/guides/stay-up-to-date/index.md

```diff
@@ -16,22 +16,7 @@ import CardGroup from "/src/components/CardGroup"
 
 <CardGroup cols={1} gap="1rem" stacked equalHeightRows={false} commonCardProps={{padding: "1.25rem"}}>
 
-<Card title="Update to 2022-12" href="/guides/stay-up-to-date/releases/2022-12" />
-<Card title="Update to 2022-11" href="/guides/stay-up-to-date/releases/2022-11" />
-<Card title="Update to 2022-10" href="/guides/stay-up-to-date/releases/2022-10" />
-<Card title="Update to 2022-09" href="/guides/stay-up-to-date/releases/2022-09" />
-<Card title="Update to 2022-08" href="/guides/stay-up-to-date/releases/2022-08" />
-<Card title="Update to 2022-07" href="/guides/stay-up-to-date/releases/2022-07" />
-<Card title="Update to 2022-06" href="/guides/stay-up-to-date/releases/2022-06" />
-<Card title="Update to 2022-05" href="/guides/stay-up-to-date/releases/2022-05" />
-<Card title="Update to 2022-04" href="/guides/stay-up-to-date/releases/2022-04" />
-<Card title="Update to 2022-03" href="/guides/stay-up-to-date/releases/2022-03" />
-<Card title="Update to 2022-02" href="/guides/stay-up-to-date/releases/2022-02" />
-<Card title="Update to 2022-01" href="/guides/stay-up-to-date/releases/2022-01" />
-<Card title="Update to 2021-12" href="/guides/stay-up-to-date/releases/2021-12" />
-<Card title="Update to 2021-11" href="/guides/stay-up-to-date/releases/2021-11" />
-<Card title="Update to 2021-10" href="/guides/stay-up-to-date/releases/2021-10" />
-<Card title="See older releases" href="/guides/stay-up-to-date/releases" />
+<!-- replaced-by-docs-sourcer-automatically-do-not-edit -->
```

Comment:
This is a bummer but we will have to face this and figure out a good way forward.

Context: late last year [Jim did a Hackday](https://gruntwork-io.slack.com/archives/CM6RR6JE7/p1668710670725539) and was able to automatically generate release notes using the docs sourcer. Unfortunately this is currently a manual process. If we merged your PR as is and released it to prod - we would blow away the index of pages that were automatically generated.

The simplest thing you can do is to just revert the change to just this file and keep in mind that as you continue working locally, you need to not commit it. 

Right now, Jim has a calendar reminder for himself to generate the monthly release notes by running that plugin manually. 

TLDR; We've got this unpolished process regarding this particular file :-/. Easiest thing for this PR is to make sure you don't check it in even though it will show up as modified when you run the docs-sourcer locally.

Reply:
Thanks for the heads up! I will update them now, but will add a TODO on the description to remember this later.